nixos/modules/security/acme.nix

```diff
@@ -136,6 +136,25 @@ let
           challenge to ensure the DNS entries required are available.
         '';
       };
+
+      mustStaple = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Turns on the OCSP Must-Staple TLS extension.
+          Make sure you know what you're doing! See:
+          https://blog.apnic.net/2019/01/15/is-the-web-ready-for-ocsp-must-staple/
+          https://blog.hboeck.de/archives/886-The-Problem-with-OCSP-Stapling-and-Must-Staple-and-why-Certificate-Revocation-is-still-broken.html
+        '';
+      };
+
+      additionalLegoRenewFlags = mkOption {
```

Comment:
I'd marginally prefer `extraLegoRenewFlags` because it's more normative for similar options in NixOS (think idiomatic `extraConfig`, `extraPackages`, or even more specific ones like `boot.loader.grub.extraEntries` or `hardware.pulseaudio.extraModules`).

Reply:
done~